examples/fargate-serverless/main.tf

```diff
@@ -152,82 +149,10 @@ data "aws_eks_addon_version" "latest" {
   most_recent        = true
 }
 
-#---------------------------------------------------------------
-# Modifying CoreDNS for Fargate
-#---------------------------------------------------------------
-
-locals {
```

Comment:
All this lives in the CoreDNS addon now so users do not need to re-create if they want to use this method of assuming control of CoreDNS with Helm